dev-duckies-sm-proj/src/pages/register-page/register.jsx

```diff
@@ -1,5 +1,122 @@
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
+
 export default function Register() {
+  const [email, setEmail] = useState('');
+  const [username, setUsername] = useState('');
+  const [password, setPassword] = useState('');
+  const [confirmPassword, setConfirmPassword] = useState('');
+
+  // Checks if the email is valid
+  const emailTest = /\S+@\S+\.\S+/.test(email);
```

Comment:
You do not need this line of code. Changing the input element `type` to `email` will automatically check if the email is valid.
```suggestion
```